docs/tutorials/get-started-with-ngo.md

```diff
@@ -0,0 +1,619 @@
+
+1. From the Unity Editor, select **Window** > **Package Manager**.
+2. From the Package Manager, select **Add (+)** > **Add package by name…**
+3. Type (or copy and paste) `com.unity.netcode.gameobjects` into the package name field, then select **Add**.
+
+Now that you have a new project and the NGO package installed, you can start creating the essential components of an NGO multiplayer game.
+
+## Add the basic components
+
+This section guides you through adding the essential components of a networked game:
+
+- [A NetworkManager component](#create-the-networkmanager-component)
+- [A player object](#create-an-object-to-spawn-for-each-connected-player)
+- [A scene](#add-your-scene-to-the-build)
+
+### Create the `NetworkManager` component
+
+This section guides you through creating a `NetworkManager` component.
+
+First, create the `NetworkManager` component:
+
+1. Right-click in the **Hierarchy** tab, then select **Create Empty** to create an empty GameObject.
+
+2. Rename the empty GameObject **NetworkManager**.
+
+3. Select **NetworkManager**, then select **Add Component** from the **Inspector** tab.
+
+4. Select **Netcode** > **NetworkManager** from the component list.
+
+5. Select **UnityTransport** as the **Network Transport**.
+6. Save the scene by selecting **File** > **Save**.
+
+### Create an object to spawn for each connected player
+
+This section guides you through creating an object that spawns for each connected player.
+
+1. In the Unity Editor, right-click within the **Hierarchy** tab, then select **3D Object** > **Capsule**.
+2. Name the Capsule Object **Player**.
+3. With Player selected, add a `NetworkObject` component in the **Inspector** tab by selecting **Add Component** > **NetworkObject**.
+4. Right-click within the **Assets** folder under the **Project** tab, then select **Create** > **Folder**.
+5. Name the folder **Prefabs**.
+6. Make the **Player** object you created earlier into a prefab by dragging it from the **Hierarchy** tab into the **Prefabs** folder.
+
+7. Delete the Player from the scene by selecting the Player capsule within the **Scene** tab, then pressing the **Delete** key.
+
+:::tip
+You can remove the `Player` object from the scene because you assign this network prefab to the Player Prefab property in the `NetworkManager` component. The library doesn't support defining a player object as an in-scene placed `NetworkObject`.
+:::
+
+8. Select **NetworkManager**.
+9. With `NetworkManager` selected, locate the **PlayerPrefabs** field in the **Inspector** tab.
+
+10. Drag the **Player** prefab from the **Project** tab into the **PlayerPrefab** slot you created in the **Inspector** tab.
+
+:::important
+
+When you drop the prefab into the **PlayerPrefab** slot, you are telling the library that when a client connects to the game, it should automatically spawn this prefab as the character for the connecting client. NGO won’t spawn a player object if you do not have any prefab set as the **PlayerPrefab**.
```

Comment:
I think we should move this to the beginning of the section, to clearly state the "why" before giving the detailed instructions. We could also directly link to this page for more info: https://docs-multiplayer.unity3d.com/netcode/current/basics/networkobject#player-objects. We could also do the same and link to specific docs pages in the other sections of this tutorial

Reply:
Consider it done!